modules/distribution/src/repository/resources/conf/validator.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Remove separation new lines. JSON we don't keep new lines.

Reply:
Refactored in master.

Reply:
Ack.